src/docs/010_manual/20_install.adoc

```diff
@@ -72,13 +76,13 @@ NOTE: `dtcw.bat` wraps the `dtcw.ps1` script and executes it in powershell. This
 
 --
 
-NOTE: In order to support development for your project on both, Linux and Windows, it makes sense to download all three scripts to your project.
-
-NOTE: if you use cmd.exe, follow the PowerShell instructions but replace `.ps1` with `.bat`.
+NOTE: In order to support development for your project on both, Linux and Windows, it makes sense to download all three scripts to your project. 
+//What do I have to do? What if I am on Windows bit my colleagues are on Linux? 
```

Comment:
good question. Best way would be that one of your colleagues installs the bash wrapper into the project. Another solution is to use WSL on windows to install it yourself